OnBaseDocsApi/Attributes/VerifyProfileHeaderAttribute.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Web.Http.Controllers;
+using System.Linq;
+
+namespace OnBaseDocsApi.Attributes
+{
+    public class VerifyProfileHeaderAttribute : BaseAttribute
+    {
+        public override void OnAuthorization(HttpActionContext actionContext)
+        {
+            if (!actionContext.Request.Headers.TryGetValues("OnBase-Profile", out var profiles))
+            {
+                // The request does not have the required header.
+                SetUnauthorizedResult(actionContext, "The account has not been granted access.");
+                return;
+            }
+
+            if (profiles.Count() != 1)
+            {
+                // The request has more than has more than one header value.
+                SetUnauthorizedResult(actionContext, "The account has not been granted access.");
+                return;
+            }
```

Comment:
The bodies of these if statements are the same. Can be combined with an OR statement

Reply:
Good point!